package.json

```diff
@@ -25,50 +25,47 @@
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-flow": "^6.23.0",
     "babel-register": "^6.26.0",
-    "babelify": "^7.3.0",
-    "browserify": "^13.3.0",
     "check-dependencies": "^1.1.0",
-    "clean-css": "^3.4.23",
+    "css-loader": "^0.28.8",
     "eslint": "^4.14.0",
     "eslint-plugin-flowtype": "^2.40.1",
-    "express": "^4.14.0",
+    "extract-text-webpack-plugin": "^3.0.2",
+    "file-loader": "^1.1.6",
     "flow-bin": "^0.62.0",
-    "glob": "^7.1.1",
     "jest": "^22.0.4",
     "jest-serializer-html": "^4.0.1",
     "js-yaml": "^3.3.1",
     "json-stable-stringify": "^1.0.1",
     "jspngopt": "^0.2.0",
     "less": "~2.7.1",
-    "less-plugin-clean-css": "^1.5.1",
-    "morgan": "^1.7.0",
+    "less-loader": "^4.0.5",
     "nomnom": "^1.8.1",
     "object-assign": "^4.1.0",
     "pako": "1.0.4",
     "pre-commit": "^1.2.2",
     "rimraf": "^2.6.2",
     "selenium-webdriver": "^2.48.2",
     "sri-toolbox": "^0.2.0",
+    "style-loader": "^0.19.1",
     "stylelint": "^8.4.0",
     "stylelint-config-standard": "^18.0.0",
-    "uglify-js": "~2.7.5",
+    "uglifyjs-webpack-plugin": "^1.1.6",
     "webpack": "^3.6.0",
     "webpack-dev-server": "^2.8.2"
   },
   "bin": "cli.js",
   "scripts": {
-    "lint": "eslint katex.js server.js cli.js webpack.config.js webpackDevServer.js src test contrib dockers && stylelint static/fonts.less static/katex.less",
+    "lint": "eslint katex.js katex.webpack.js cli.js webpack.common.js webpack.config.js webpackDevServer.js src test contrib dockers && stylelint static/fonts.less static/katex.less",
     "flow": "flow",
     "jest": "jest",
     "jest-update": "jest --updateSnapshot",
     "coverage": "jest --coverage",
-    "copy": "cp -a static/. build/ && cp -a contrib build/",
     "clean": "rm -rf build/* node_modules/",
     "clean-install": "npm run clean && npm i",
     "test": "check-dependencies && npm run lint && npm run flow && npm run jest",
-    "build-css": "lessc --clean-css static/katex.less build/katex.css",
-    "prestart": "rimraf build && npm run build-css && npm run copy",
-    "start": "check-dependencies && node webpackDevServer.js",
+    "start": "webpack-dev-server --open --config webpackDevServer.js",
+    "build": "webpack",
+    "watch": "webpack --watch",
```

Comment:
I wonder if we should use `--watch` for `start` to maintain the same behavior?  The current server supports hot reloading (except for the CSS which was kind of annoying).

Reply:
Sorry, `webpack-dev-server` does hot reloading, everything is fine.